clab/clab.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/srl-labs/containerlab/runtime/docker"
 	"github.com/srl-labs/containerlab/runtime/ignite"
 	"github.com/srl-labs/containerlab/types"
+	"k8s.io/utils/strings/slices"
```

Comment:
our `internal/slices/slices.go` contains the `Contains` function already. No need for this import

Reply:
migrated to utils.StringInSlice